apps/ui/src/Communities/CommunitiesFeed.tsx

```diff
@@ -22,12 +22,17 @@ export interface CommunitiesFeedProps {
 const CommunitiesFeed: React.FC<CommunitiesFeedProps> = ({ communities }) => {
   const { user } = useUser();
   const userCommunitiesStatusOriginal = useUserCommunitiesStatus(
-    user?.id ?? 'admin',
+    user?.id ?? '1',
   );
 
   const [userCommunitiesStatus, setUserCommunitiesStatus] = useState(
     userCommunitiesStatusOriginal,
   );
+
+  useEffect(() => {
+    setUserCommunitiesStatus(userCommunitiesStatusOriginal);
```

Comment:
didn't we speak about removing this?

Reply:
I need it for the sake of creating a community here.. Maybe when I'm done with connecting and creating a community I'll check if I still need it?